src/utils/LocaleUtils.ts

```diff
@@ -2,7 +2,7 @@ import {LocalizedLanguages} from '../i18n';
 import {DeviceUtils} from './DeviceUtils';
 
 const localizedText = (data: LocalizedLanguages) => {
-  const locale = DeviceUtils.locale.substring(0, 2);
+  const locale = DeviceUtils.locale?.substring(0, 2);
```

Comment:
Check why locale is undefined sometimes

Reply:
Changed the Locale util function by following this answer.
[Ios Locale issue](https://stackoverflow.com/a/47349998)